PiwikTracker.php

```diff
@@ -1288,6 +1288,25 @@ public function setRequestTimeout($timeout)
     }
 
     /**
+     * If a proxy is needed to look up the address of the Piwik site, set it with this
+     * @param $proxy
+     * @param int $proxyPort
+     */
+    public function setProxy($proxy, $proxyPort = 80){
+        $this->proxy = $proxy;
+        $this->proxyPort = $proxyPort;
+    }
+
+    public function getProxy(){
+        if(isset($this->proxy) && isset($this->proxyPort)){
```

Comment:
Do you mind adding some spaces between `if(` eg `if (` and before the opening bracket eg `)) {`.

`if (isset($this->proxy) && isset($this->proxyPort)) {`